content/en/docs/concepts/configuration/secret.md

```diff
@@ -1244,83 +1244,34 @@ these pods.
 
 ## Information security for Secrets
 
-Although ConfigMap and Secret work similarly, Kubernetes applies some additional
-protection for Secret objects.
+Although ConfigMaps and Secrets work similarly, Kubernetes applies some
+additional protections to Secret objects.
 
-Secrets often hold values that span a spectrum of importance, many of which can
-cause escalations within Kubernetes (e.g. service account tokens) and to
-external systems. Even if an individual app can reason about the power of the
-Secrets it expects to interact with, other apps within the same namespace can
-render those assumptions invalid.
+Secrets often hold values that span a spectrum of importance. Many of these
+values can cause escalations within Kubernetes and to external systems. Even if
+an individual app can reason about the power of the Secrets it expects to
+interact with, other apps within the same namespace can render those assumptions invalid.
```

Comment:
What does this reformatting mean to localization teams? It means they have to compare each and every line of the text because that is how git diff works. As it is shown here, the whole paragraph looks rewritten -- 5 lines removed and 4 lines added. The localization team has to check carefully which word has been changed. The net effect is the   whole paragraph has to be retranslated.

Do we really want to do this, with a compelling reason? Are these changes bring us any value?

Please stop massaging the text in this way. There are tons of topics we have not covered in the docs and there are several hundred open issues to be closed. Could you look into some of those issues instead of reediting the existing content in this way?

Reply:
You're right, I had this PR out for review before our discussion in Slack and in other PRs about larger changes like this. I'll revert some of the tangential changes that aren't strictly related to this PR and split the commits for the new page to make it easy to see where content was changed vs just moved.

Reply:
@tengqm reverted most changes to existing Secrets concept (except for one clarity change). Split the creation of the new page into multiple commits so that the diffs are clear about what changed vs what moved. 

@PushkarJ @aramase @ritazh the content changes are identical to the previous state, nothing new to review :)